src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc

```diff
@@ -0,0 +1,34 @@
+[id="adding-custom-certificates-to-trust-store_{context}"]
+= Adding custom public SSL certificates to {prod-short} trust-store
+
+This procedure describes how to configure {prod-short} to be able to handle HTTP requests from untrusted sources.
+
+.Procedure
+. Save the certificate that you want to apply to `ca.crt` file.
+. Create a new *configMap* with the certificate:
++
+[subs="+quotes"]
+----
+$ kubectl create configmap custom-public-cert --from-file=ca.crt -n=che
+----
+. Set the `global.customPublicCert` property to `true`.
++
+=====
+.If {prod-short} is deployed using a link:https://helm.sh/[Helm Chart]
+. Clone the https://github.com/eclipse/che[che] project
+. Go to `deploy/kubernetes/helm/che` directory
+. Update the `global.tls.useCustomPublicCerts` property. To do that, add the following option to the `helm upgrade` command:
++
+----
+$ helm upgrade che --set global.tls.useCustomPublicCerts=true --set global.ingressDomain=$(minikube ip).nip.io .
```

Comment:
Is it only about minikube?

Reply:
Could you please refer to that place?

Reply:
https://github.com/eclipse/che-docs/blob/che-16002/src/main/pages/che-7/administration-guide/proc_deploying-registries-in-kubernetes.adoc#L20

Reply:
fixed